fighthealthinsurance/models.py

```
@@ -1783,12 +1790,67 @@ class Denial(ExportModelOperationsMixin("Denial"), models.Model):  # type: ignor
+        return amount_to_int(self.billed_amount_cents)
+
+    def get_allowed_cents(self) -> typing.Optional[int]:
+        return amount_to_int(self.allowed_amount_cents)
+
+    def get_paid_cents(self) -> typing.Optional[int]:
+        return amount_to_int(self.paid_amount_cents)
+
+    def set_billed_cents(self, value: typing.Optional[int]) -> None:
+        # setattr (vs direct assignment) keeps mypy happy without the
+        # django-stubs plugin, which is also how CI runs in some envs.
+        setattr(self, "billed_amount_cents", amount_to_str(value))
```

Ruff (0.15.12)

[warning] Do not call `setattr` with a constant attribute value. It is not any safer than normal property access.

Replace `setattr` with assignment

(B010)